packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -1191,4 +1191,83 @@ describe('<lion-fieldset>', () => {
       });
     });
   });
+
+  describe('disabled', () => {
+    describe('capture', () => {
+      let el;
+      beforeEach(async () => {
+        el = await fixture(html`
+          <${tag} name="data" disabled>
+            <${tag} name="full_name" disabled>
+              <${childTag} .modelValue="${'John'}" name="first_name" disabled></${childTag}>
+              <${childTag} .modelValue="${'Doe'}" name="last_name" disabled></${childTag}>
+            </${tag}>
+            <${childTag} .modelValue="${'Amsterdam'}" name="city" disabled></${childTag}>
+          </${tag}>
+        `);
+      });
+
+      it('does stuff', () => {
+        // when
+        const result = el.capture();
+        // then
+        expect(result).to.eql({
+          full_name: {
+            first_name: 'John',
+            last_name: 'Doe',
+          },
+          city: 'Amsterdam',
+        });
+      });
+    });
+
+    describe('restore', () => {
+      let el;
+      let fullNameEl;
+      let firstNameEl;
+      let lastNameEl;
+      let cityEl;
+      beforeEach(async () => {
+        el = await fixture(html`
+          <${tag} name="data" disabled>
+            <${tag} name="full_name" disabled>
+              <${childTag} name="first_name" disabled></${childTag}>
+              <${childTag} name="last_name" disabled></${childTag}>
+            </${tag}>
+            <${childTag} name="city" disabled></${childTag}>
+          </${tag}>
+        `);
+        fullNameEl = el.querySelector('[name="full_name"]');
+        firstNameEl = el.querySelector('[name="first_name"]');
+        lastNameEl = el.querySelector('[name="last_name"]');
+        cityEl = el.querySelector('[name="city"]');
+      });
+
+      it('does stuff', () => {
+        // when
+        el.restore({
+          full_name: {
+            first_name: 'John',
+            last_name: 'Doe',
+          },
+          city: 'Amsterdam',
+        });
+        // then
+        expect(el.modelValue).to.eql({
```

Comment:
Looks good. 

I think this would actually give back nothing, since modelValue should not give back disabled values.
But if you replace this with a capture() or do modelValue with (at least some) none disabled on top, it would be great